src/main/java/oss/fosslight/controller/SearchController.java

```diff
@@ -0,0 +1,177 @@
+
+
+    @GetMapping(value=SEARCH.LICENSE)
+    public @ResponseBody ResponseEntity<Object> saveLicenseSearchFilter(LicenseMaster licenseMaster
+            , HttpServletRequest req
+            , HttpServletResponse res){
+
+        int page = Integer.parseInt(req.getParameter("page"));
+        int rows = Integer.parseInt(req.getParameter("rows"));
+        String sidx = req.getParameter("sidx");
+        String sord = req.getParameter("sord");
+
+        licenseMaster.setCurPage(page);
+        licenseMaster.setPageListSize(rows);
+        licenseMaster.setSortField(sidx);
+        licenseMaster.setSortOrder(sord);
+
+        searchService.saveLicenseSearchFilter(licenseMaster, loginUserName());
+
+        return makeJsonResponseHeader("saved");
+
+    }
+
+
+    @GetMapping(value=SEARCH.OSS)
+    public @ResponseBody ResponseEntity<Object> saveOssSearchFilter(OssMaster ossMaster,
+                                                                    HttpServletRequest req){
+
+        int page = Integer.parseInt(req.getParameter("page"));
+        int rows = Integer.parseInt(req.getParameter("rows"));
+        String sidx = req.getParameter("sidx");
+
+        if(sidx != null) {
+            sidx  = sidx.split("[,]")[1].trim();
+        }
+
+        ossMaster.setSidx(sidx);
+        ossMaster.setCurPage(page);
+        ossMaster.setPageListSize(rows);
+
+        List<String> values = Arrays.asList("https://", "http://", "www.");
+        String homepage =req.getParameter("homepage");
+        if(!values.contains(homepage)){
+            homepage = homepage.replaceFirst("^((http|https)://)?(www.)*", "");
+            ossMaster.setHomepage(homepage);
+        }
+        ossMaster.setSearchFlag(CoConstDef.FLAG_YES);
+        ossMaster.setSearchFlag(CoConstDef.FLAG_YES);
+
+        searchService.saveOssSearchFilter(ossMaster, loginUserName());
+
+
+        return makeJsonResponseHeader("saved");
+    }
+
+
+    @GetMapping(value=SEARCH.PARTNER)
+    public @ResponseBody ResponseEntity<Object> savePartnerSearchFilter(PartnerMaster partnerMaster,
+                                                                        HttpServletRequest req){
+
+        int page = Integer.parseInt(req.getParameter("page"));
+        int rows = Integer.parseInt(req.getParameter("rows"));
+        String sidx = req.getParameter("sidx");
+        String sord = req.getParameter("sord");
+
+        partnerMaster.setCurPage(page);
+        partnerMaster.setPageListSize(rows);
+        partnerMaster.setSortField(sidx);
+        partnerMaster.setSortOrder(sord);
+
+        if(partnerMaster.getStatus() != null) {
+            String statuses = partnerMaster.getStatus();
+            if(!isEmpty(statuses)) {
+                String[] arrStatuses = statuses.split(",");
+                partnerMaster.setArrStatuses(arrStatuses);
+            }
+        }
+
+        searchService.savePartnerSearchFilter(partnerMaster, loginUserName());
+        return makeJsonResponseHeader();
```

Comment:
Some method don't reponse saved, Others response with saved
```java
return makeJsonResponseHeader("saved");
  ```
If there is no concrete reason, please align it